crates/rolldown_common/src/types/symbol_ref_db.rs

```diff
@@ -74,13 +74,6 @@ impl SymbolRefDbForModule {
     SymbolRef::from((self.owner_idx, symbol_id))
   }
 
-  /// This method is used to hide the `SymbolTable::create_symbol` method since
-  /// `SymbolRefDbForModule` impl `Deref` for `SymbolTable`.
-  #[deprecated = "Use `create_facade_root_symbol_ref` instead"]
-  pub fn create_symbol(&mut self) {
```

Comment:
It's usued intentionally to prevent developers wrongly from calling `SymbolTable::create_symbo`.

Reply:
I've searched the codebase and couldn't find any usage of this function. I believe it's safe to remove.